db/seeds.rb

```diff
@@ -61,23 +61,32 @@
 shipping_offices = JSON.parse(File.read(Rails.root.join('db', 'seeds', 'shipping_offices.json')))
 
 shipping_offices.each do |office|
-  ShippingOffice.create(office)
+  ShippingOffice.create(office.except('location').merge(location_attributes: office['location']))
```

Comment:
So this is a fun trick now that "locatable" models have [`accepts_nested_attributes_for :locations`](http://api.rubyonrails.org/classes/ActiveRecord/NestedAttributes/ClassMethods.html). When creating a shipping office (or any other locatable), you can create and save a location by tacking on a `location_attributes` hash in one fell swoop.

This line takes the "location" hash (brought along for the ride from the JSON files) and _basically_ renames it to work with `accepts_nested_attributes_for`.

I think there are other opportunities to use this elsewhere in the app in a future PR.